readme.txt

```diff
@@ -120,6 +120,7 @@ Currently, the following add-ons are available for Event Tickets:
 = [4.10.2] TBD =
 
 * Tweak - Use buttons instead of links and add better feedback on checkin (disable buttons) [70618]
+* Tweak - remove empty "Primary Info" column from attendee list email and export [122274]
```

Comment:
I have to clean up this before release, be sure to capitalize each entry, thanks!

```suggestion
* Tweak - Remove empty "Primary Info" column from attendee list email and export [122274]
```